src/mmw/js/src/core/templates/header.ejs

```diff
@@ -13,7 +13,7 @@
                     </a>
                     <ul class="dropdown-menu">
                         <li><a href="#">Profile</a></li>
-                        <li><a href="accounts/logout">Logout</a></li>
+                        <li><a href="/user/logout" class="user-logout" onclick="return false;">Logout</a></li>
```

Comment:
Agreed.  I think event handlers should be implemented in js files, not html/templates, so it's easier to find and read the behavior.